index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<title>Portfolio</title>
+</head>
+<body>
+	<section class="screen_container">
+		<header>
+			<nav>
+				<section class="toolbar_container">
+					<p class="Title_logo">My Logo</p>
+					<img src="./images/i.png"  class="logo" alt="Icon menu">
+				</section>
+			</nav>
+		</header>
+		
+		<main class="main_container">
+			<section class="text_container">
+				<p class="paragraphe">Hey There. <br/>I’m Leader.</p>
```

Comment:
- [ ] Please use `h1` tag for your name. If you want that google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.